kroxylicious-test-tools/src/main/java/io/kroxylicious/proxy/config/MicrometerDefinitionBuilder.java

```diff
@@ -17,7 +18,15 @@ public MicrometerDefinitionBuilder(String type) {
 
     @Override
     protected MicrometerDefinition buildInternal(String type, Map<String, Object> config) {
-        var configType = MicrometerConfigurationHookContributorManager.getInstance().getConfigType(type);
+        Class<? extends BaseConfig> result;
+        try {
+            result = ContributionManager.INSTANCE.getDefinition(MicrometerConfigurationHookContributor.class, type, (clazz, typeName) -> "").configurationType();
+        }
+        catch (IllegalArgumentException e) {
+            // Catch and re-throw with a more user-friendly error message
+            throw new IllegalArgumentException("No micrometer configuration hook found for name '" + type + "'");
```

Comment:
ditto on this, can the errorGenerator in `getDefinition` produce this IllegalArgumentException instead.

Reply:
Yeah.

Reply:
fixed in ec1c649b79f516f46e01d0991948a8bc05556932